src/plugin/file/kml/ui/kmlnode.js

```diff
@@ -400,6 +408,16 @@ class KMLNode extends SlickTreeNode {
    */
   setImage(image) {
     this.image_ = image;
+    if (this.image_) {
+      // Give the image style control options.
+      var options = {};
+      options[ControlType.COLOR] = ColorControlType.PICKER_RESET;
+      this.image_.setLayerOptions(options);
```

Comment:
This might be more appropriate in `readGroundOverlay_` in the KML parser, where the layer is created.